package.json

```diff
@@ -176,7 +176,7 @@
 							"type": "string"
 						},
 						"default": [],
-						"description": "Glob expressions for files to be ignored by the project parser"
+						"description": "Glob expressions for files to be ignored by the project parser (should match the file and not folder! i.e. **/generated/**/* instead of **/generated/*)"
```

Comment:
So the glob has to match the whole path of the file, and does not hit when the parent folder is ignored?
I do not think this is optimal. i would prefer if this works as in gitgnore.
i. if I ignor a folder all children will be automatically ignored

Reply:
https://git-scm.com/docs/gitignore#_pattern_format
I believe exactly emulating the gitignore pattern would require us to implement our own glob matching...
But, yes we could also apply it to folders. I will have a look at this